bindings/rust/s2n-tls/src/config.rs

```diff
@@ -317,6 +323,14 @@ impl Builder {
         Ok(self)
     }
 
+    /// Sets whether or not default system certificates will be loaded into the trust store. Set to
+    /// false for increased performance if system certificates are not needed during certificate
+    /// validation.
```

Comment:
Rust doc comments should be a one line summary followed by more info below if needed.

```suggestion
    /// Sets whether or not default system certificates will be loaded into the trust store.
    ///
    /// Set to false for increased performance if system certificates are not needed during
    /// certificate validation.
```